sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -317,7 +317,7 @@ def LargestPerKey(pcoll, n, key=None):
 
   @staticmethod
   @ptransform.ptransform_fn
-  def SmallestPerKey(pcoll, n, key=None, reverse=True):
+  def SmallestPerKey(pcoll, n, *, reverse=True, key=None):
```

Comment:
given we force KW-only arguments, we can move key before reverse, same as we do in other combiners.